src/components/CountriesList.js

```diff
@@ -0,0 +1,23 @@
+import { NavLink } from 'react-router-dom';
+
+function CountryList(props) {
+  return props.countries.map((element) => {
+    return (
+      <div className="list-group" key={element._id}>
+        <a
+          className="list-group-item list-group-item-action"
```

Comment:
Here it would be better using a `NavLink `or a `Link `from the `react-router-dom` so the page doesn't reload :)